backend/src/nodes/nodes/image_filter/blur_bilateral.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+import cv2
+import numpy as np
+
+from . import category as ImageFilterCategory
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import ImageInput, NumberInput
+from ...properties.outputs import ImageOutput
+from ...utils.utils import get_h_w_c
+
+
+@NodeFactory.register("chainner:image:bilateral_blur")
+class BlurNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Apply surface/bilateral blur to an image."
+        self.inputs = [
+            ImageInput(),
+            NumberInput("Radius", controls_step=1),
+            NumberInput("Color Sigma", controls_step=1, default=75, maximum=100),
+            NumberInput("Space Sigma", controls_step=1, default=75, maximum=100),
```

Comment:
There isn't actually a maximum for these values, and the documentation even mentions large values as >150. There's definitely a point around 300-400 where it starts approximating Gaussian too much to be very useful, but I think we should let the max be higher than 100. Maybe the 250-300 range.